src/main/java/com/es/phoneshop/model/product/ArrayListProductDao.java

```diff
@@ -70,7 +69,27 @@ public synchronized void delete(Long id) {
 
     @Override
     public synchronized List<Product> sort(List<Product> unsortedProducts, String productField, String order){
-        return new ArrayList<>();
+        if (productField != null && !productField.isEmpty()) {
+            return unsortedProducts.stream()
+                    .sorted(getComparator(productField, order))
+                    .collect(Collectors.toList());
+        } else {
+            return unsortedProducts;
+        }
+    }
+
+    private Comparator<Product> getComparator(String productField, String order) {
+        comparators = new HashMap<>();
+        comparators.put("price", Comparator.comparing(Product::getPrice));
```

Comment:
Here and in other places: consider defining string values as constants (Ctrl + Alt + C in Intellij IDEA) for better readability and to avoid typos in case the value may be used more than once in future.